components/footer/index.tsx

```diff
@@ -50,6 +50,14 @@ const Footer: React.FC = () => (
               Staff Applications
             </a>
           </div>
+          <div >
```

Comment:
does having this div help with the styling at all, we can avoid endless div wrapper hell by removing divs that aren't actually adding anything to whats inside of them or its not grouping more than one thing inside of it. try seeing if having the a tag by itself messes anything up. This applies to the other single wrapped divs in this file even though we did not write this originally

Reply:
does nothing, removed